pkg/jtracer/jtracer_test.go

```diff
@@ -14,18 +14,4 @@
 
 package jtracer_test
 
-import (
-	"testing"
-
-	"github.com/opentracing/opentracing-go"
-	"github.com/stretchr/testify/assert"
-
-	"github.com/jaegertracing/jaeger/pkg/jtracer"
-)
-
-func TestOT(t *testing.T) {
-	mockTracer := opentracing.NoopTracer{}
-	jtracer := jtracer.OT(opentracing.NoopTracer{})
-
-	assert.Equal(t, mockTracer, jtracer.OT)
-}
+// TODO
```

Comment:
remove this file, otherwise it will cause 0% coverage to be reported.

Reply:
when I did this, I was getting linting error.

Reply:
that's because you didn't have .nocover file

Reply:
It's an intentional safety check.

Reply:
when test file is not there, linting causes an error. `No jtracer_test.go`

Reply:
did you commit the deletion? The list of files for linting may be read from git

Reply:
That's correct. I did not.